drivers/mongodb/connection.go

```diff
@@ -21,13 +21,13 @@ func NewConnection(uri string) (*mongo.Client, error) {
 		log.Fatalf("Failed to connect to MongoDB: %s", err)
 	}
 
-	defer func() {
-		if err = client.Disconnect(ctx); err != nil {
-			log.Fatalf("Failed disconnecting MongoDB connection: %s", err)
-		}
-	}()
+	// defer func() {
+	// 	if err = client.Disconnect(ctx); err != nil {
+	// 		log.Fatalf("Failed disconnecting MongoDB connection: %s", err)
+	// 	}
+	// }()
```

Comment:
Team is there an special reason of why we are commenting these lines of code, instead of removing them? Because this is dead-code. Let's bear in mind that we move out the client.Disconnect to be implemented externally.

Reply:
That's right, we move the disconnect out of the function. I am going to remove the dead code.